.github/workflows/compilation.yml

```diff
@@ -31,6 +31,15 @@ jobs:
       run: |
         brew update
         brew install gettext texinfo bison flex gnu-sed gsl gmp mpfr
+    
+    - name: Install in MSYS2
+      if: matrix.os[0] == 'windows-latest'
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: MINGW32
+        install: base-devel git make texinfo flex bison patch binutils mingw-w64-i686-gcc mingw-w64-i686-dlfcn mingw-w64-i686-mpc
```

Comment:
Is there a reason for choosing 32 bit packages instead of the mingw-w64-x86_64 packages?

Reply:
I'm not sure if i'm correct but it's like you need to install packages according on their architecture to properly work, I will try to change that and test it if you have doubt, that's okay to me

Reply:
if you check on their msys2 packages, they have pattern of architectures to use the packages here: https://packages.msys2.org/package/

Reply:
My point is that you use mingw-w64-i686-gcc here instead of mingw-w64-x86_64-gcc for building. The GitHub Action runs on a 64 bit system.

Reply:
ahh got it, because i think that mingw-w64-x86_64-* packages will go to mingw64 environment when building the toolchain and cannot find the packages because the mingw32 is using the mingw32 environment when finding packages

Reply:
like the libusb package, no package of libusb in msys environment but there's a libusb in mingw64 or mingw32 environment

Reply:
Okay, I've merged it now.